platforms/substrate/charts/substrate-key-mgmt/templates/job.yaml

```diff
@@ -167,27 +167,35 @@ spec:
           echo "Generate nodekey"
           node_id=$($command key generate-node-key --file node_key 2>&1)
 
-          echo "Generate Account key and store it in json file"
-          $command key generate --scheme Sr25519 --output-type json >> accKeygenOutput.json
-          ACC_SEED=$(jq -r '.secretSeed' accKeygenOutput.json)
-          ACC_ADDR=$(jq -r '.ss58Address' accKeygenOutput.json)
+          echo "Generate aura key"
+          $command key generate --scheme Sr25519 --output-type json >> auraKeygenOutput.json
+          AURA_SEED=$(jq -r '.secretSeed' auraKeygenOutput.json)
+          AURA_ADDR=$(jq -r '.ss58Address' auraKeygenOutput.json)
 
-          {{- if eq .Values.node.type "validator" }}
-            echo "Generate separategrandpa key for validator" 
-            $command key generate --scheme Ed25519 --output-type json >> grandpaKeygenOutput.json 
-            GRANDPA_SEED=$(jq -r '.secretSeed' grandpaKeygenOutput.json)
-            GRANDPA_ADDR=$(jq -r '.ss58Address' grandpaKeygenOutput.json)
+          echo "Generate grandpa key"
+          $command key generate --scheme Ed25519 --output-type json >> grandpaKeygenOutput.json
+          GRANDPA_SEED=$(jq -r '.secretSeed' grandpaKeygenOutput.json)
+          GRANDPA_ADDR=$(jq -r '.ss58Address' grandpaKeygenOutput.json)
+
+          {{- if eq .Values.node.type "member" }}
+            echo "Generate Account key for member/owner node"
+            $command key generate --scheme Sr25519 --output-type Json >> accKeygenOutput.json          
+            ACCOUNT_SEED=$(jq -r '.secretSeed' accKeygenOutput.json)
+            ACCOUNT_ADDR=$(jq -r '.ss58Address' accKeygenOutput.json)
 
             echo "
               {
                 \"data\": {
                 \"node_id\": \"$node_id\",
                 \"node_key\": \"$(cat node_key)\",
-                \"aura_seed\": \"$ACC_SEED\",
-                \"aura_addr\": \"$ACC_ADDR\",
+                \"aura_seed\": \"$AURA_SEED\",
+                \"aura_addr\": \"$AURA_ADDR\",
                 \"grandpa_seed\": \"$GRANDPA_SEED\",
                 \"grandpa_addr\": \"$GRANDPA_ADDR\",
```

Comment:
Do we need grandpa and aura keys for member nodes?

Reply:
I have kept them to be same as what DC did. Additional account keys are generated for member nodes, thats all.